main.py

```diff
@@ -92,6 +92,8 @@ def check_origin(self, origin):
 
 def main():
     application = tornado.web.Application([
+        (r"/rocon_web_remocon/(.*)",tornado.web.StaticFileHandler,{"path":"./www/rocon_web_remocon", "default_filename":"index.html"}),
+        (r"/rocon_web_common/(.*)",tornado.web.StaticFileHandler,{"path":"./www/rocon_web_common", "default_filename":"index.html"}),
```

Comment:
@adamantivm I can move the RosbridgeProxyHanlder to /ws, but if I create only one file handler, then we should change some things in the files, beacause I think it originally worked with two different handlers.